lib/screens/release_health/release_health_view_model.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:redux/redux.dart';
+import 'package:sentry_mobile/redux/actions.dart';
+import 'package:sentry_mobile/redux/state/app_state.dart';
+import 'package:sentry_mobile/types/organization.dart';
+import 'package:sentry_mobile/types/project.dart';
+import 'package:sentry_mobile/types/release.dart';
+
+class ReleaseHealthViewModel {
+  ReleaseHealthViewModel(this._store, this._organization, this.project, this.releases, this.loading);
+  final Store<AppState> _store;
+  final Organization _organization;
+
+  final Project project;
+  List<Release> releases;
+  bool loading;
```

Comment:
should we make them private or read-only getters?

Reply:
Made the properties final, as they only change when the class is inited from state.